### PR TITLE
Added GitHub action editorconfig

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
Added GitHub action editorconfig checker

### Why is it needed?
To check compliance with the editorconfig rules when sending PR

https://github.com/editorconfig-checker/editorconfig-checker

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/pull/15538 https://github.com/modxcms/revolution/issues/14311

### CODE REVIEW

**Comment by JoshuaLuckers** [comment]
Can't we add this to the existing workflow?

**Comment by Ibochkarev** [comment]
> Can't we add this to the existing workflow?

What's wrong with a single action?

**Comment by JoshuaLuckers** [comment]
A waste of resources, also two places to maintain the OS etc.

**Comment by Ibochkarev** [comment]
> Can't we add this to the existing workflow?

Done

File: .ecrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "exclude": ["\\.swf$", "\\.css$"]
```
Comment (JoshuaLuckers) [inline .ecrc:2]: If there are any `.swf` files in the core we should remove it.

**Review by theboxer — APPROVED** [review]

**Review by Mark-H — APPROVED** [review]

**Review by Mark-H — COMMENTED** [review]
Funnily enough this PR itself is preventing it from getting merged 🙈

**Comment by opengeek** [comment]
Is this still needed? Isn't this covered by the phpcs checks?

**Comment by Ibochkarev** [comment]
> Is this still needed? Isn't this covered by the phpcs checks?

phpcs only checks the php format, it checks the other for compliance. But after adding phpcs, it may not be so relevant already

**Pull request closed without merging**